src/pages/_Components/WhatStacksCanDo/WhatStacksCanDo.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+import Picker from "../Picker/Picker";
+import styles from "../../styles.module.css";
+import customStyle from "./WhatStacksCanDo.module.css";
+
+const text = {
+    clientFeedbackMessage: '"Ensono Stacks made our transformation significantly easier and gave us confidence right from day one" - '
+}
+
+const WhatStacksCanDo = () => {
+    const baseUrl = useBaseUrl("docs/");
+
+    return (
+        <div>
+            <span className={styles.primaryColour}><i style={{display: 'inline-block'}} className={styles.textCenter}>{text.clientFeedbackMessage}<strong>Ensono Client</strong></i></span>
```

Comment:
Do we want to pull `style={{display: 'inline-block'}}` out into a css file?

Reply:
done